Automattic-Tracks-iOS/Event Logging/EventLogging+Sentry.swift

```diff
@@ -1,6 +1,11 @@
 import Foundation
 import Sentry
 
+public enum EventLoggingErrorType {
+    case fatal
+    case debug
```

Comment:
The name `Sentry` uses for this is `error`. Considering the implementation is domain agnostic, it's totally fine (maybe even better) to keep this naming scheme, but I wanted to mention this in case you had other thoughts.

Reply:
Which part are you referring to? The enum name, or one of the cases? :)

Reply:
This was a comment on the `case debug` line. Github's UI in that regard is not super clear. I also got confused there for a second about which lines I added the comment for. The only difference between multiline comment and a single line comment seems to be the extra line: `Comment on lines 41 to 45`.

Reply:
Ah yeah – I prefer `debug` in this case because it's not necessarily an error – we often send Sentry events for non-error states (like...this thing happened we didn't expect...)